charts/ingress-nginx/Chart.yaml

```diff
@@ -1,8 +1,6 @@
 annotations:
   artifacthub.io/changes: |
-     - "[helm] Support custom port configuration for internal service (#9846)"
-     - "Adding resource type to default HPA configuration to resolve issues with Terraform helm chart usage (#9803)"
-     - "Update Ingress-Nginx version controller-v1.7.1"
+     - "Use controller-v1.7.1"
```

Comment:
isn't this unrelated change?

Reply:
Yes,
These are the changes that opensource community has done in this version I believe, do we need to preserver that information?